src/lib/components/header/header-dashboard/HeaderDashboard.ViewModel.ts

```diff
@@ -154,11 +153,7 @@ export class HeaderDashboardViewModel {
     const response = await this.workspaceService.fetchWorkspaces(userId);
     let isAnyWorkspaceActive: undefined | string = undefined;
     const data = [];
-    if (
-      response?.isSuccessful &&
-      response?.data?.data &&
-      response?.data?.data?.length > 0
-    ) {
+    if (response?.isSuccessful && response?.data?.data) {
```

Comment:
destructure this beforehand and then use it in if statement like
const {isSuccessful,data} = response;